u2f-chrome-extension/debug.js

```diff
@@ -0,0 +1,92 @@
+/**
```

Comment:
This points out a bug in the upstream source, this really has no reason to belong here. I'll send a cl to fix, then perhaps you could re-export.